translate/cloud-client/translate_v3_batch_translate_text_with_glossary_test.py

```diff
@@ -12,18 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import backoff
 import os
 import pytest
 import uuid
 import translate_v3_batch_translate_text_with_glossary
 import translate_v3_create_glossary
 import translate_v3_delete_glossary
+
+from google.api_core.exceptions import DeadlineExceeded, GoogleAPICallError, RetryError
+from google.cloud.exceptions import NotFound
 from google.cloud import storage
 
 PROJECT_ID = os.environ["GCLOUD_PROJECT"]
 GLOSSARY_INPUT_URI = "gs://cloud-samples-data/translation/glossary_ja.csv"
 
 
+@backoff.on_exception(
+    backoff.expo, (DeadlineExceeded, GoogleAPICallError,
+                   RetryError), max_time=60
```

Comment:
I may be wrong, but `RetryError` is thrown when the automatic retry failed, then what's the use of retrying?

Reply:
> I'm not sure how `backoff` works exactly when you decorate the pytest's fixture.
> If you think it is the right way to do it, please explain how it works.

I have no idea, I usually write samples in Java. I am not expert in Python. Please guide me thru.

Reply:
Should I just remove RetryError ?